build-dbt-cross-compiler/BUILD_go/aarch64-elf/BUILD.go

```diff
@@ -0,0 +1,45 @@
+package aarch64
```

Comment:
I don't like mixing naming conventions, re: underscore and hyphens in the paths.